### DISCUSSION TITLE
Discord rich presence not working

### DISCUSSION THREAD

**Discussion by Zockerboi** [ppy/osu#27140] [Q&A] [answered]
### Type

Cosmetic

### Bug description

Well, as the title says, discord rich presence does not show the stants and stuff. 

### Screenshots or videos

![image](https://github.com/ppy/osu/assets/85823953/d59bed6d-6ae0-4495-bbc0-3cf22e6e260d)
![image](https://github.com/ppy/osu/assets/85823953/c2c42a0a-cb04-4bd5-93ab-4a923dac616a)

### Version

2024.131.0-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14230203/compressed-logs.zip)

**Comment by Joehuu** [comment]
Did you set your status in osu! to "Appear offline". That's the only other condition that'll turn it off:

https://github.com/ppy/osu/blob/b0420f7ed9e945cb69ffd844664df0705ccc4e8f/osu.Desktop/DiscordRichPresence.cs#L89-L93

**Reply by Zockerboi** [reply]
No, i have do not disturb on.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/pull/27354
